plugin-build/src/main/kotlin/io/sentry/android/gradle/extensions/LogcatExtension.kt

```diff
@@ -0,0 +1,21 @@
+package io.sentry.android.gradle.extensions
+
+import io.sentry.android.gradle.instrumentation.logcat.LogcatLevel
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Property
+import javax.inject.Inject
+
+open class LogcatExtension @Inject constructor(objects: ObjectFactory) {
+    /**
+     * Enables or disables the Logcat feature.
+     * Defaults to true.
+     */
+    val enabled: Property<Boolean> = objects.property(Boolean::class.java).convention(true)
```

Comment:
Since this feature causes more data to be sent, we should probably disable it by default. Maybe we can enable it by default with the next major bump.

Reply:
If we do the same for Timber I'm fine with keeping it enabled 🚀